### PR TITLE
Added some leftout html tags 

### PR DESCRIPTION
**Pull request by kautuk98**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [![Circle CI](https://circleci.com/gh/JabRef/JabFox.svg?style=svg)](https://circleci.com/gh/JabRef/JabFox)
+![Forks](https://img.shields.io/github/forks/JabRef/JabFox.svg?style=svg)
```
Comment (tobiasdiez) [inline README.md]: I don't think this adds real additional value as github already displays the number of forks. So please remove this.

Comment (kautuk98) [reply README.md]: Thank you for the review.

**Comment by kautuk98** [comment]
Done. Thanks.

**Comment by kautuk98** [comment]
> Thanks for the quick follow-up. I'll merge now 
> 
> I would like to continue working with you. What do you think about solving #51?

Sure, I would take a look at #51

**Comment by kautuk98** [comment]
Does this merge count as +1 for the hacktoberfest?

**Comment by kautuk98** [comment]
@tobiasdiez am i eligible to get some swags.

**Comment by koppor** [comment]
@kautuk98 This counts for the Digital Ocean Hacktoberfest. I think, you registered at https://hacktoberfest.digitalocean.com/, didn't you?

JabRef will send out a sticker. Just send your shipping address to me. Find my email at my profile.

**Comment by kautuk98** [comment]
I didn't get it. Doesn't this is mentioned in the Hacktoberfest that we can get swag and stickers from other communities too? That is so kind of you. Thank you for stickers.

**Pull request merged**